app/reducers/index.js

```diff
@@ -10,16 +10,45 @@ const defaultState = {
       created: 1488984810654,
       avatar: 'sprout',
       status: 0,
+      currentRating: 7,
       ratings: [],
     },
     {
       id: 1,
       name: 'Drink more champagne',
       created: 1488984810658,
-      avatar: 'sprout',
+      avatar: 'pumpkin',
+      status: 0,
+      currentRating: 8,
+      ratings: [],
+    },
+    {
+      id: 2,
+      name: 'Be cool',
+      created: 1488984810658,
+      avatar: 'pepper',
       status: 0,
+      currentRating: 10,
       ratings: [],
     },
+    // {
```

Comment:
Ok, you and @shiryz should work together to have a better way of injecting mock data, once you have the server and the client working together